runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -295,7 +295,8 @@ typedef struct LocalJ9UTF8Buffer {
 	/**
 	 * Constructs an empty LocalJ9UTF8Buffer.
 	 */
-	LocalJ9UTF8Buffer() = default;
+	LocalJ9UTF8Buffer() : utf8(nullptr), capacity(0), cursor(nullptr)
+	{}
```

Comment:
I'd like to see other improvements to this local type:
* this is C++ code so the `typedef` is unwanted
* constructors should use field initializers consistently
* `const` is not significant for a parameter `const char c`
* I'm not sure the constructor `LocalJ9UTF8Buffer(char *buffer, size_t length)` is a good idea (the cast would be better at the call site where it is likely clearer that it's reasonable)

Reply:
Thanks for the feedback.

Reply:
> constructors should use field initializers consistently

To be clear, your preference here is member initializer lists for constructors similar to how the empty constructor is now, rather than the assignment in the constructor body?